lib/jekyll-compose/arg_parser.rb

```diff
@@ -24,4 +25,8 @@ def title
   def force?
     !!options["force"]
   end
+
+  def source
+    source = config['source'].gsub(/^#{Regexp.quote(Dir.pwd)}/, '')
```

Comment:
Oh I hadn't noticed that. I'm pretty confident those do nothing, would you remove all three?